content-test/components/SiteIcon.test.js

```diff
@@ -104,3 +114,35 @@ describe("SiteIcon", () => {
     });
   });
 });
+
+describe("SiteIcon loaded", () => {
+  let node;
+  let instance;
+
+  function setup(customProps = {}, customSiteProps = {}) {
+    const site = Object.assign({}, fakeProps.site, customSiteProps);
+    const props = Object.assign({}, fakeProps, customProps, {site});
+    node = document.createElement("div");
+    instance = ReactDOM.render(<SiteIcon {...props} />, node);
+  }
+
+  afterEach(() => {
+    ReactDOM.unmountComponentAtNode(node);
+  });
+
+  it("should use fallback on small 1x1 favicon", done => {
+    setup({}, {favicon_url: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAAEElEQVR42gEFAPr%2FAP%2F%2F%2F%2F8J%2BwP9vTv7fQAAAABJRU5ErkJggg%3D%3D"});
+    instance.refs.favicon.addEventListener("load", () => {
+      assert.isTrue(instance.state.showFallback);
+      done();
+    });
+  });
+
+  it("should use favicon for 16x16 favicon", done => {
+    setup({}, {favicon_url: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAAAAAA6mKC9AAAAD0lEQVQYV2P4jwYYRrYAAID5%2FwHnXPpAAAAAAElFTkSuQmCC"});
```

Comment:
same as above, you can move this one into the 'favicon' test (https://github.com/mozilla/activity-stream/blob/master/content-test/components/SiteIcon.test.js#L75) to re-use setup function